src/NBomber/DomainServices/NBomberRunner.fs

```diff
@@ -35,15 +37,26 @@ let runSession (testInfo: TestInfo) (context: NBomberContext) (dep: IGlobalDepen
         return nodeStats
     }
 
-let run (appType: ApplicationType) (context: NBomberContext) =
+let private canProgressBarBeCreated() =
```

Comment:
rename to getApplicationType() and make sure that this function retuns ApplicationType and not boolean flag